### PR TITLE
Auto-rename blank CSV columns. refs #10115

### PR DESCRIPTION
**Pull request by mcantelon**
When transforming/importing CSV, give blank header columns names so column
values, etc., can more easliy be referenced.

### CODE REVIEW

File: lib/QubitFlatfileImport.class.php [hunk 1]
```diff
@@ -468,6 +468,25 @@ private function handleByteOrderMark($fh)
...
   {
+    // Assign names to blank columns
+    $baseLabel = 'Untitled';
+    $labelNumber = 1;
+    foreach ($this->columnNames as $index => $name)
+    {
+      if (empty($name))
+      {
+        // Increment label number if column already exists
+        while(in_array($baseLabel . $labelNumber, $this->columnNames))
```
Comment (MikeFE) [inline lib/QubitFlatfileImport.class.php:484]: not sure if it's worth breaking out into a helper function since it's only several lines, but generating a new column name seems outside the scope of handleColumnRenaming?

Comment (mcantelon) [inline lib/QubitFlatfileImport.class.php:484]: Hrm, yeah true... I'll do that.

**Comment by mcantelon** [comment]
Merged.
